bitbots_vision/docs/tutorials/yolo_open_vino.rst

```diff
@@ -0,0 +1,63 @@
+=========
+OpenVINO
+=========
+
+Convert Darknet to OpenVINO models
+==================================
+
+Install OpenVINO
+----------------
+You can skip this step, if you have OpenVINO installed or access to a robot.
```

Comment:
Why to a robot?
Do you mean to a compute stick?

Reply:
Because the robots have OpenVINO installed.
Now it is just Amy, in the future every robot needs it.